Argentum20.vbp

```diff
@@ -158,7 +159,7 @@ VersionComments="www.ao20.com.ar"
 VersionCompanyName="Argentum20"
 VersionLegalCopyright="Argentum20 Staff"
 VersionProductName="Argentum20 by Noland Studios"
-CondComp="Developer = 1 : Compresion = 0 : DEBUGGING = 1 : PYMMO = 0"
+CondComp="Developer = 1 : Compresion = 0 : DEBUGGING = 1 : PYMMO = 0 : DISABLE_SUBCLASSING = 1"
```

Comment:
What's this about?

Reply:
this is a tricky one, vb6 doesn't let us handle mouse wheel events by default, to do this we need to use "subclasing" feature which let you read the windows events sent to a window, but it require to properly unload the feature when you close a window

Because debug app run inside vb6 environment when you stop a debug app it doesn't unload properly and crash the whole environment so I added this flag to disable the feature to use use while develop unless I need it, disabling it makes the mouse wheel not work on the ui